Geoparsing Text with the Edinburgh Geoparser - Lesson migration

ianmilligan1 opened the pull request:
Staging our newest lesson for deployment. [Continues the review ticket that you can find here](https://github.com/programminghistorian/ph-submissions/issues/26).

ianmilligan1 committed:
Initial check-in of @bea-alex' lesson
Geoparsing Text with the Edinburgh Geoparser
Now to run checks and see how it shakes out

ianmilligan1 commented:
Ping @bea-alex – please follow along here. We'll check everything, and then begin a soft launch so you can proof things before it goes out to the world.

ianmilligan1 committed:
fixing YAML for geoparsing lesson

ianmilligan1 commented:
@bea-alex: Here's a proof here: <https://deploy-preview-633--ph-dev.netlify.com/lessons/geoparsing-text-with-edinburgh>. Could you review and let me know how it works? I re-worked the figure syntax and I think it all moved over well, but if you could pay particular attention.

mdlincoln committed:
Typo in abstract

ianmilligan1 commented on lessons/geoparsing-text-with-edinburgh.md:
```diff
@@ -15,7 +15,7 @@ difficulty: 3
 activity: presenting
 topics: [mapping]
-abstract: "This tutorial teaches users how to use the Edinburgh Geoparse to process a piece of text, extract and resolve the locations contained within it, and plot them as a web map."
+abstract: "This tutorial teaches users how to use the Edinburgh Geoparser to process a piece of text, extract and resolve the locations contained within it, and plot them as a web map."
```
Good catch, thanks @mdlincoln

bea-alex commented:
Hi,

I spotted two things:

In the Download section, 2nd para:

geoparser-march2016-2.tar.gz
-> geoparser-march2016.tar.gz

In the Credits and Citation section:

 ADD URL HERE -> change that to the url which the lesson will be accessible at

I wasn’t sure whether to make those changes in the original lesson or whether it’s ported somewhere.  I can make them myself if I just need to update the lesson I worked on previously.

Best,

Bea



> On 24 Oct 2017, at 15:35, Ian Milligan <notifications@github.com> wrote:
> 
> @bea-alex <https://github.com/bea-alex>: Here's a proof here: https://deploy-preview-633--ph-dev.netlify.com/lessons/geoparsing-text-with-edinburgh <https://deploy-preview-633--ph-dev.netlify.com/lessons/geoparsing-text-with-edinburgh>. Could you review and let me know how it works? I re-worked the figure syntax and I think it all moved over well, but if you could pay particular attention.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/programminghistorian/jekyll/pull/633#issuecomment-339011845>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGK8H1ZI8Wg4g_agKG66uLPhF2zC80wCks5svfW8gaJpZM4QEbgt>.
> 


The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.

ianmilligan1 committed:
Updating download filename and lesson URL
Responding to @bea-alex comments on #633

ianmilligan1 commented:
I added them in (probably easiest for me to update the pull request at this point). If you're feeling happy with it, I think we're ready to go. Maybe we can schedule for a Monday morning (EST) launch, afternoon over in your neck of the woods?

bea-alex commented:
Sounds great.  I’ll have another read over it at the weekend just in case.

Thanks for all you help.

Bea

> On 26 Oct 2017, at 13:29, Ian Milligan <notifications@github.com> wrote:
> 
> I added them in (probably easiest for me to update the pull request at this point). If you're feeling happy with it, I think we're ready to go. Maybe we can schedule for a Monday morning (EST) launch, afternoon over in your neck of the woods?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/programminghistorian/jekyll/pull/633#issuecomment-339649853>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGK8HxqvmOr0mb0J6fTvhKoW0C6pqnvDks5swHsjgaJpZM4QEbgt>.
> 


The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.

mdlincoln committed:
include an automatically-generated ToC

This hasn't been documented in the editor guidelines yet; I'll be
working on that re: #641

mdlincoln commented on lessons/geoparsing-text-with-edinburgh.md:
```diff
@@ -0,0 +1,385 @@
+*	Geo-parsing multiple text files, and
+*	Extracting geo-resolution output to TSV.
+
+### Prerequisites and Terminology
```
The levels of every header after `Introduction` should be fixed up so they're more logical. Right now, every one is an `h3` or `h4`, and so the generated ToC doesn't end up being very useful. Generally, each section should be at `h2`, with subsections incremented as needed.

ianmilligan1 replied:
Thanks @mdlincoln – started this long enough ago I forgot about the ToC. I'll rationalize it.

ianmilligan1 committed:
Changing heading levels so ToC more legible
Updates pull request #633

mdlincoln committed:
set Introduction section to h2

ianmilligan1 committed:
Updating lesson date to next Monday for launch

ianmilligan1 commented:
Great, thanks for the tweaks and review @mdlincoln. I've set the date for next Monday, @bea-alex, so we can do any last minute tweaks or changes. Just let me know, otherwise I'll merge this when I get to work on Monday. 😄

ianmilligan1 commented:
Turns out Monday was a wash but in the absence of any further comments, I'm going to update the date on this and merge.

Thanks all!

ianmilligan1 committed:
updating publication date
